packages/yourloops/app/error.tsx

```diff
@@ -0,0 +1,117 @@
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import * as React from "react";
+import { useTranslation } from "react-i18next";
+
+import { useTheme } from '@material-ui/core/styles';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import TextField from '@material-ui/core/TextField';
+
+import {
+  STORAGE_KEY_SESSION_TOKEN,
+  STORAGE_KEY_TRACE_TOKEN,
+  STORAGE_KEY_USER,
+} from "../lib/auth/hook";
+
+import sendMetrics from "../lib/metrics";
+
+interface OnErrorProps {
+  event: Event | string;
+  source?: string;
+  lineno?: number;
+  colno?: number;
+  error?: Error;
+}
+
+function OnError(props: OnErrorProps): JSX.Element {
+  const { t } = useTranslation("yourloops");
+  const theme = useTheme();
+  const [showMore, setShowMore] = React.useState(false);
+  const fullScreen = useMediaQuery(theme.breakpoints.down('md'));
+
+  React.useEffect(() => {
+    try {
+      sendMetrics("app-crash", props);
+    } catch (err) {
+      console.error(err);
+    }
+  }, [props]);
+
+  const handleOK = () => {
+    // Logout the user
+    sessionStorage.removeItem(STORAGE_KEY_SESSION_TOKEN);
+    sessionStorage.removeItem(STORAGE_KEY_TRACE_TOKEN);
+    sessionStorage.removeItem(STORAGE_KEY_USER);
+    // Then reload the page
+    window.location.reload();
+  };
+
+  const handleShowMore = () => {
+    setShowMore(true);
+  };
+
+  let moreInfos: JSX.Element | null = null;
+  if (showMore) {
+    const error = props.error ? `Error: ${props.error.message}\nStack: ${props.error.stack}` : "N/A";
+    const info = `${props.event.toString()}\nSource: ${props.source}:${props.lineno}:${props.colno}\n${error}`;
+    moreInfos = (
+      <React.Fragment>
+        <DialogContentText style={{ marginTop: "1em", marginBottom: "0px" }} color="textSecondary">{t("app-crash-info")}</DialogContentText>
+        <TextField id="dialog-app-crash-technical-info" style={{ width: "100%" }} inputProps={{ color: "grey" }} color="secondary" multiline rows={4} value={info}/>
+      </React.Fragment>
+    );
+  } else {
+    moreInfos = (
+      <Button id="dialog-app-button-more-info" variant="text" color="primary" style={{ marginTop: "1em" }} onClick={handleShowMore}>
+        {t("app-crash-button-more-info")}
+      </Button>
+    );
+  }
+
+  return (
+    <Dialog id="dialog-app-crash" open={true} fullScreen={fullScreen} fullWidth={true} maxWidth="sm">
+      <DialogTitle id="dialog-app-crash-title">{t("app-crash-title")}</DialogTitle>
+      <DialogContent>
+        <DialogContentText id="dialog-app-crash-explanation" color="textPrimary">{t("app-crash-text")}</DialogContentText>
```

Comment:
do we want to make translations for `app-crash-text` ?

Reply:
Already done in previous PR